### PR TITLE
Update class for tag component

### PR DESCRIPTION
**Pull request by sBenjamin270495**
### What does this PR do?

Update tag class for the new tag component

Related to https://github.com/upfluence/backlog/issues/1684

### What are the observable changes?
TagScreen

### Good PR checklist

- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated documentation
- [x] Properly labeled

### CODE REVIEW

File: less/base/_tags.less [hunk 1]
```diff
@@ -12,46 +12,38 @@
   display: flex;
   align-items: center;
   justify-content: center;
-
-  padding: 0 @spacing-xxx-sm;
-  border-radius: @default-radius;
+  height: var(--spacing-px-24);
+  border-radius: 999px;
```
Comment (Miexil) [inline less/base/_tags.less:16]: Hmmmm 🤔 really ?

File: less/core-v2/_variables.less [hunk 2]
```diff
@@ -15,6 +15,7 @@
   // Spacers — Because sometimes we need to let elements breathe
   //
   --spacing-px-3: 3px;
+  --spacing-px-9: 9px;
```
Comment (Miexil) [inline less/core-v2/_variables.less:18]: Are we sure we want to add this one ?
We are "supposed" to work with a 6px scaling grid. The px-3 was already a weird one to add, but we needed it in multiple cases.
If you need to use 9px in the tag component only, I suggest you just put 9px directly

File: less/base/_tags.less (hunk 1 above)
Comment (Miexil) [reply less/base/_tags.less:16]: `12px` or `14px` seems to have the same outcome. Please don't use random high values 🙏

**Review by phndiaye — COMMENTED** [review]
@JulienVannier66 @Miexil already said everything so good for me beside those comments. GG on the first PR 🥳

File: less/base/_tags.less (hunk 1 above)
Comment (sBenjamin270495) [reply less/base/_tags.less]: border-radius max is 12px is good

**Review by JulienVannier66 — APPROVED** [review]
LGTM GG!

**Pull request merged**